spec/models/draw_spec.rb

```diff
@@ -255,10 +255,23 @@
 
   describe '#next_groups' do
     it 'calls NextGroupQuery' do
-      draw = build_stubbed(:draw)
+      draw = FactoryGirl.build_stubbed(:draw)
       allow(NextGroupsQuery).to receive(:call).with(draw: draw)
       draw.next_groups
       expect(NextGroupsQuery).to have_received(:call).with(draw: draw)
     end
   end
+
+  describe '#notify_next_groups' do
+    # rubocop:disable RSpec/ExampleLength
+    it 'sends a selection invite to the leaders of the next groups' do
+      draw = FactoryGirl.build_stubbed(:draw)
+      group = instance_spy('Group', leader: instance_spy('User'))
+      allow(draw).to receive(:next_groups).and_return([group])
+      mailer = instance_spy('StudentMailer')
```

Comment:
ummm... changing this to `instance_spy('student_mailer')` passes this test... not sure what's going on. I'll leave it to you to update and rebase.